### ISSUE TITLE
NTR: History of excessive licorice use (General history, localization & modifiers)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7834]
New term request History of excessive licorice use (elthat_101030015825): Glycyrrhizic acid, liquorice  UMLS: 2074637

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000162
- label: Excessive licorice consumption
- parents: [Nutrition history (HP:4000123)](https://hpo.jax.org/app/browse/term/HP:4000123)
- pmid: PMID:35321787|PMID:23185686
- def: Ingestion of large amounts of licorice.
- comment: Licorice is a popular sweetener found in many soft drinks, food products, snacks and herbal medicines. Licorice toxicity is characterized by the symptom triad of hypertension, hypokalemia, and metabolic alkalosis, so-called pseudohyperaldosteronism. The active ingredient in licorice is glycyrrhizin, which is converted to 3beta-monoglucuronyl-18beta-glycyrrhetinic acid (3MGA) and 18beta-glycyrrhetinic acid (GA) in the gut. These active metabolites inhibit the 11beta-hydroxysteroid dehydrogenase type II enzyme and prevent the conversion of cortisol to cortisone. Unlike cortisone, cortisol binds to the mineralocorticoid receptor. This cortisol excess leads to increased potassium and proton excretion as well as to sodium and water retention, with increased systemic resistance. The triad of symptoms typical of classic hyperaldosteronism (Conn syndrome) results from the aldosterone-like effect: hypokalemia, hypertension, and metabolic alkalosis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7834
- synonyms:
